bin/build_utils

```diff
@@ -25,3 +25,25 @@ function gen_versions() {
     echo $version
   done
 }
+
+function is_release_commit() {
+
+  # fetching tags is required for git_description to work
+  git fetch --tags
+  git_description=$(git describe)
+
+  if [ "$git_description" != "v${VERSION}" ]
+  then
+    true
+  else
+    false
+  fi
+}
+
+function append_commit_hash_to_version() {
```

Comment:
looks like this function should be called `git_tag_suffix`

Reply:
also - it is better to have this function above the other one as it is using it. it will read beter

Reply:
done